common/s2ecmd/s2ecmd.cpp

```diff
@@ -48,13 +48,15 @@
 typedef int (*cmd_handler_t)(int argc, const char **args);
 
 typedef struct _cmd_t {
-    char *name;
+    const char *name;
```

Comment:
Does this have to be a cpp file? You should still be able to compile c and cop files together. I can’t see anything here that is really c++ specific

Reply:
I think it's actually better to compile in c++ mode even c files. That's how it found the const bug above.